packages/clang5.rb

```diff
@@ -0,0 +1,46 @@
+require 'package'
+
+class Clang5 < Package
+  description 'C language family frontend for LLVM.'
+  homepage 'http://clang.llvm.org/'
+  version '5.0.1'
+  source_url 'http://releases.llvm.org/5.0.1/cfe-5.0.1.src.tar.xz'
+  source_sha256 '135f6c9b0cd2da1aff2250e065946258eb699777888df39ca5a5b4fe5e23d0ff'
+
+  depends_on 'buildessential'
+  depends_on 'cmake'
+  depends_on 'llvm5'
+  depends_on 'clang5_libcpp'
+  depends_on 'clang5_openmp'
+  depends_on 'llvm5_compiler_rt'
+  depends_on 'llvm5_polly'
+  depends_on 'llvm5_unwind'
+  depends_on 'lld5'
+
+  def self.build
+    system "mkdir -p mybuilddir"
+    Dir.chdir "mybuilddir" do
+ 	    system "cmake -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release .."
+        system "cmake --build . --  -j#{CREW_NPROC}"
+    end
+  end
+
+  def self.install
+    Dir.chdir "mybuilddir" do
+      system "cmake -DCMAKE_INSTALL_PREFIX=#{CREW_DEST_DIR}/usr/local -P cmake_install.cmake"
+      FileUtils.cd("#{CREW_DEST_PREFIX}/bin") do
+        system "echo '#!/bin/bash' > clang++w"
```

Comment:
Probably shouldn't use special characters in filenames.  Can you use `clangppw` instead?